extensions/interactions/GraphInput/directives/oppia-interactive-graph-input.component.ts

```diff
@@ -38,17 +38,21 @@ import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
   styleUrls: []
 })
 export class InteractiveGraphInput implements OnInit, OnDestroy {
-  @Input() graphWithValue: string;
-  @Input() canAddVertexWithValue: string;
-  @Input() canDeleteVertexWithValue: string;
-  @Input() canMoveVertexWithValue: string;
-  @Input() canEditVertexLabelWithValue: string;
-  @Input() canAddEdgeWithValue: string;
-  @Input() canDeleteEdgeWithValue: string;
-  @Input() canEditEdgeWeightWithValue: string;
-  @Input() lastAnswer: null | GraphAnswer = null;
-
-  graph: GraphAnswer;
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion. For more information, see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  @Input() graphWithValue!: string;
+  @Input() canAddVertexWithValue!: string;
+  @Input() canDeleteVertexWithValue!: string;
+  @Input() canMoveVertexWithValue!: string;
+  @Input() canEditVertexLabelWithValue!: string;
+  @Input() canAddEdgeWithValue!: string;
+  @Input() canDeleteEdgeWithValue!: string;
+  @Input() canEditEdgeWeightWithValue!: string;
+  // Last answer is null if graph is submitted for the first time.
+  @Input() lastAnswer!: null | GraphAnswer;
```

Comment:
Usually we put `null` last, why is it different here?

Reply:
Nothing special, just typo mistake.